--- a/src/pose_utils/qcprot.cpp	2018-02-23 19:42:05.586924284 +0000
+++ b/src/pose_utils/qcprot.cpp	2018-02-23 19:42:20.511054061 +0000
@@ -172,7 +172,7 @@
     const double   *fx2 = coords2[0], *fy2 = coords2[1], *fz2 = coords2[2];
     double          G1 = 0.0, G2 = 0.0;
 
-    A[0] = A[1] = A[2] = A[3] = A[4] = A[5] = A[6] = A[7] = A[8] = A[9] = 0.0;
+    A[0] = A[1] = A[2] = A[3] = A[4] = A[5] = A[6] = A[7] = A[8] = 0.0;
 
     if (weight != NULL)
     {
